policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/age_amount/max_amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Saskatchewan provides the following maximum amount under the age amount credit.
+values: 
+  2022-01-01: 5_061
+  2023-01-01: 5_380
+
+metadata:
+  unit: currency-CAD
+  label: Saskatchewan age amount credit max amount
+  reference:
+  - title: 2022 Saskatchewan Personal Tax Credit Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-22e.pdf
+  - title: 2023 Saskatchewan Personal Tax Credit Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1sk/td1sk-23e.pdf
+  - title: The Income Tax Act, 2000 DIVISION 2 Section 17, Page 15, Age Credit 17(1)
```

Comment:
Since this specifies $4,894 for 2017-20 and adjusts for inflation from there, could you add the 2021 value so we can include the history since 2017? In general please tie or explain values in references to the yaml.